### DISCUSSION TITLE
backstrap

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#535] [Q&A (Help)] [answered]
Hi all, i'm using last backpack 5.x, can use backstrap classes in Mail Blades ?
If yes, how to ?

Or, can i install boostrap 5.3 and use it without conflicts ?

Thanks

**Comment by promatik** [comment]
Hi @blondie63, emails are very "sensitive", you cannot use libraries on it.

With emails, you must embed all the necessary css on it, you can try to embed all the bootstrap framework on it, but most of the things will not work.

The best approach is really to keep it simple and do the css on your own 🤷‍♂️

**Reply by blondie63** [reply]
Thanks

**Comment by o15a3d4l11s2** [comment, marked as answer]
As an additional suggestion: you could start with some templates from https://litmus.com/community/templates or any other place, just keep in mind that there are a lot of templates out there that are really not so well done - they look fine in one web client, but are very much broken in others.

Another good idea is to use something like https://mailtrap.io - it helps with testing emails and also validates them for compatibility with many of the widely used email clients.

And last - what @promatik said - start simple, keep the styles embedded into the page, preferably directly as a `style` attribute on every DOM element.

**Reply by blondie63** [reply]
Thanks !!
